app/models/miq_report/generator/trend.rb

```diff
@@ -35,11 +35,8 @@ def build_results_for_report_trend(options)
       trend_klass = db_options[:trend_db].kind_of?(Class) ? db_options[:trend_db] : Object.const_get(db_options[:trend_db])
 
       time_range_cond = ["timestamp BETWEEN ? AND ?", start_time, end_time]
-      recs = VimPerformanceDaily.find(:all,
-                                      :conditions  => VimPerformanceDaily.merge_conditions(where_clause, time_range_cond),
-                                      :include     => includes,
-                                      :ext_options => {:class => trend_klass, :only_cols => only_cols, :reflections => associations, :tz => tz, :time_profile => time_profile}
-                                     )
+      ext_options = {:class => trend_klass, :only_cols => only_cols, :reflections => associations, :tz => tz, :time_profile => time_profile}
+      recs = VimPerformanceDaily.includes(includes).where(where_clause).where(time_range_cond).find(:all, :ext_options => ext_options)
```

Comment:
@matthewd This confuses me.

1. This is using `:ext_options`, which is for `ActsAsActiveRecordModel`.
2. `VimPerformanceDaily` extends `MetricRollup`
3. `MetricRollup` extends `ActiveRecord::Base`

Questions

1. Is this `ActiveRecord` or `AAARM`
2. Is this refactor safe?
3. Is `:ext_options` a bug?